<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #D2ECFA</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblVideo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>1024</width>
      <height>1024</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px; background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>30</y>
      <width>601</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; border-radius: 10px</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>331</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>80</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Points:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPoints">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>201</width>
       <height>231</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>80</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>310</y>
      <width>601</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; border-radius: 10px</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>331</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>60</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Reps Left:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblReps">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>201</width>
       <height>231</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>80</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnDone">
    <property name="geometry">
     <rect>
      <x>1288</x>
      <y>830</y>
      <width>601</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1E80C4; border-radius: 10px; color: white;</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
